app/components/MemberForm/MemberForm.tsx

```diff
@@ -61,175 +68,209 @@ const resolver = createResolver<FormSchema>({
-                  )}
-                />
-                <TextInput
-                  wrapperProps={createTestAttr(ids.nationalIdInput)}
-                  error={errors.nationalId?.message}
-                  className={classes.textInput}
-                  {...register('nationalId')}
-                  placeholder={t.nationalIdInput.placeholder}
-                  label={`${t.nationalIdInput.label}:`}
-                />
-
-                <Controller
-                  name="gender"
-                  control={control}
-                  render={({ field }) => (
-                    <Select
-                      wrapperProps={createTestAttr(ids.genderInput)}
-                      data={genders.map(v => ({
-                        value: v,
-                        label: messages.genders[v],
-                      }))}
-                      label={`${t.genderInput.label}:`}
-                      placeholder={t.selectInputs.placeholder}
-                      error={errors.gender?.message}
-                      {...field}
-                    />
-                  )}
-                />
-                <DateInput
-                  wrapperProps={createTestAttr(ids.dobInput)}
-                  className={classes.dateInput}
-                  rightSection={
-                    <CalendarIcon stroke="currentColor" size={16} />
-                  }
-                  label={`${t.dobInput.label}:`}
-                  sx={theme => ({
-                    direction: 'ltr',
-                    color: theme.colors.secondaryDefault[6],
-                  })}
-                  locale="fa"
-                  placeholder={t.selectInputs.placeholder}
-                />
 
-                <Controller
-                  name="religion"
-                  control={control}
-                  render={({ field }) => (
-                    <Select
-                      wrapperProps={createTestAttr(ids.religionInput)}
-                      data={religions.map(v => ({
-                        value: v,
-                        label: messages.religions[v],
-                      }))}
-                      placeholder={t.selectInputs.placeholder}
-                      label={`${t.religionInput.label}:`}
-                      error={errors.religion?.message}
-                      {...field}
-                    />
-                  )}
-                />
-              </SimpleGrid>
+  return (
+    <DashboardCard
+      left={
+        <ActionIcon onClick={toggle}>
+          <ArrowDownIcon width="16" height="16" color="black" />
+        </ActionIcon>
+      }
+      right={
+        <Group spacing={10}>
+          <Title order={4} color="fgDefault" weight="bold">
+            {!isEditableMode ? (
+              <>
+                {initialMember?.name} {initialMember?.surname}
+              </>
+            ) : (
+              <>
+                {watchAllFields.name} {watchAllFields.surname}
+              </>
+            )}
```

Comment:
```suggestion
            {name} {surname}
```

since the initial member is set as the default for the form we don't need to reference it here, only the form data